bundles/org.openhab.binding.freeathomesystem/src/main/java/org/openhab/binding/freeathomesystem/internal/util/FidTranslationUtils.java

```diff
@@ -0,0 +1,366 @@
+    public static final int FID_NEXT_SONG_SENSOR = 0x0093; // Next song
+    public static final int FID_PREVIOUS_SONG_SENSOR = 0x0094; // Previous song
+    public static final int FID_HOME_APPLIANCE_SENSOR = 0x0095; // Home appliance sensor
+    public static final int FID_HEAT_SENSOR = 0x0096; // Heat sensor
+    public static final int FID_ZONE_SWITCHING = 0x0097; // Zone switching
+    public static final int FID_SECURE_AT_HOME_FUNCTION = 0x0098; // Button function
+    public static final int FID_COMPLEX_CONFIGURATION = 0x0099; // Advanced configuration
+    public static final int FID_DOMUS_CENTRAL_UNIT_BASIC = 0x009A; // secure@home Central Unit Basic
+    public static final int FID_DOMUS_REPEATER = 0x009B; // Repeater
+    public static final int FID_DOMUS_SCENE_TRIGGER = 0x009C; // Remote scene control
+    public static final int FID_DOMUSWINDOWCONTACT = 0x009D; // Window sensor
+    public static final int FID_DOMUSMOVEMENTDETECTOR = 0x009E; // Movement Detector
+    public static final int FID_DOMUSCURTAINDETECTOR = 0x009F; // External IR Sensor Curtain
+    public static final int FID_DOMUSSMOKEDETECTOR = 0x00A0; // Smoke Detector
+    public static final int FID_DOMUSFLOODDETECTOR = 0x00A1; // Flood Detection
+    public static final int FID_PANEL_SUG_SENSOR = 0x00A3; // Sensor for air-conditioning unit
+    public static final int FID_TWO_LEVEL_HEATING_COOLING_ACTUATOR = 0x00A4; // Two-point controller for heating or
+                                                                             // cooling
+    public static final int FID_PANEL_THERMOSTAT_CONTROLLER_SLAVE = 0x00A5; // Slave thermostat
+    public static final int FID_WALLBOX = 0x00A6; // Wallbox
+    public static final int FID_PANEL_WALLBOX = 0x00A7; // Wallbox
+    public static final int FID_DOOR_LOCK_CONTROL = 0x00A8; // Door lock control
+    public static final int FID_VRV_GATEWAY = 0x00AA; // Room temperature controller with fan speed level
+
+    public static final int FID_SCENE_TRIGGER = 0x4800; // Scene trigger
+    public static final int FID_RULE_SWITCH = 0x4A00; // Scene trigger
+
+    // FID added based on tests
+    public static final int FID_AIRQUALITYSENSOR_PRESSURE = 0x0E017;
+    public static final int FID_AIRQUALITYSENSOR_CO2 = 0x0E018;
+    public static final int FID_AIRQUALITYSENSOR_CO = 0x0E019;
+    public static final int FID_AIRQUALITYSENSOR_NO2 = 0x0E01A;
+    public static final int FID_AIRQUALITYSENSOR_O3 = 0x0E01B;
+    public static final int FID_AIRQUALITYSENSOR_PM10 = 0x0E01C;
+    public static final int FID_AIRQUALITYSENSOR_PM25 = 0x0E01D;
+    public static final int FID_AIRQUALITYSENSOR_VOC = 0x0E01E;
+    public static final int FID_AIRQUALITYSENSOR_HUMIDITY = 0x0B03F;
+
+    public static final int FID_MOVEMENT_DETECTOR_FLEX = 0x1090;
+    public static final int FID_DIMMING_ACTUATOR_FLEX = 0x1810;
+
+    static {
+        Map<String, String> mapDesc = new HashMap<String, String>();
```

Comment:
```suggestion
        Map<String, String> mapDesc = new HashMap<>();
```
(see #16249)

You can also initialize it directly like this:
https://github.com/openhab/openhab-addons/blob/e62bcf5db4f3e3126f60ea467d31cbdd9755ee49/bundles/org.openhab.binding.miele/src/main/java/org/openhab/binding/miele/internal/handler/WashingMachineChannelSelector.java#L138-L145

I.e.:
```java
private Map<String, String> mapDesc = Map.ofEntries(Map.entry("0x0000", "Control element"));
```

etc.